substrate/substrate-test-runtime/src/lib.rs

```diff
@@ -925,6 +925,18 @@ cfg_if! {
                 }
             }
 
+            impl sp_objects::ObjectsApi<Block> for Runtime {
+                fn validated_object_call_hashes() -> Vec<Hash> {
+                    Vec::new()
+                }
+
+                fn extract_block_object_mapping(
+                    _block: Block, _successful_calls: Vec<Hash>
+                ) -> subspace_core_primitives::objects::BlockObjectMapping {
+                    subspace_core_primitives::objects::BlockObjectMapping::default()
```

Comment:
I know we admire the explicitness, but personally, I'd choose the simplicity `Default::default()` in such a case. Feel free to ignore though :P